cd/va-enp-api-task-definition.json

```diff
@@ -18,8 +18,8 @@
       },
       "portMappings": [
         {
-          "containerPort": 6011,
-          "hostPort": 6011
```

Comment:
So this needs to have a different port than the flask app?

Reply:
I was unsure if it needed to or not, but I figured I would just have it on a different port. This might be something worth talking about as a team especially in the context of the communication between API and ENP.

Reply:
Question answered in standup. We'll go with port 6012.